src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/ModellingGroupController.kt

```diff
@@ -31,6 +35,12 @@ open class ModellingGroupController(
         return modellingGroupRepository.getModellingGroupDetails(groupId)
     }
 
+    fun getContextUserModellingGroups() : List<ModellingGroup>
+    {
+        val userName = context.username!!
+        return groupLogic.getUserModellingGroups(userName)
```

Comment:
can we go with `getModellingGroupsForUser`?

Reply:
Yep, and will make url /user/modelling-groups